demisto_sdk/commands/init/tests/contribution_converter_test.py

```diff
@@ -611,7 +611,8 @@ def test_format_converted_pack(self, contribution_converter, mocker, new_pack):
                               side_efect=self.mock_format_manager())
         contribution_converter.format_converted_pack()
 
-        assert result.call_args[1].get('prev_ver') == 'xsoar-contrib/master'
+        assert result.call_args[1].get('include_untracked')
+        assert not result.call_args[1].get('interactive')
```

Comment:
Is this scenario still true? 
        Scenario: Formatting the added/modified files by checking against "xsoar-contrib/master" repo

Reply:
Please update the given / when/ then of this test